packages/core/src/utils/fileUtils.ts

```diff
@@ -310,9 +310,24 @@ export async function processSingleFileContent(
         }
         llmTextContent += formattedLines.join('\n');
 
+        // By default, return nothing to streamline the common case of a successful read_file.
+        let returnDisplay = '';
+        if (contentRangeTruncated) {
+          returnDisplay = `Read lines ${
+            actualStartLine + 1
+          }-${endLine} of ${originalLineCount} from ${relativePathForDisplay} (truncated`;
```

Comment:
wonder if we really need the word truncated unless some lines were shortened.
If that is not the case then showing the range we read should be sufficient.